src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -234,8 +234,8 @@ onUnmounted(() => {
 });
 
 onMounted(() => {
-  let cookiePeriod = cookies.get("all_deleted_messages_period");
-  if (typeof cookiePeriod === "undefined" || cookiePeriod === "") {
+    let cookiePeriod = cookies.get("all_deleted_messages_period");
+    if (typeof cookiePeriod === "undefined" || cookiePeriod === "" || cookiePeriod === null) {
```

Comment:
This will cover all three checks https://developer.mozilla.org/en-US/docs/Glossary/Falsy

```suggestion
    if (!cookiePeriod) {
```